prime-dotnet-webapi-tests/Unit Tests/SiteStatusStateEngine.Tests.cs

```diff
@@ -21,13 +21,31 @@ public static IEnumerable<object[]> SiteRegistrationActionData()
 
         [Theory]
         [MemberData(nameof(SiteRegistrationActionData))]
-        public void TestAllowableStatusChange_Active(SiteRegistrationAction action)
+        public void TestAllowableStatusChange_Actived(SiteRegistrationAction action)
+        {
+            // Arrange
+            var expectedAllowed = new[]
+            {
+                SiteRegistrationAction.Restore,
+            }.Contains(action);
+
+            // Act
+            var allowed = SiteStatusStateEngine.AllowableStatusChange(action, SiteStatusType.Archived);
+
+            // Assert
+            Assert.Equal(expectedAllowed, allowed);
+        }
+
+        [Theory]
+        [MemberData(nameof(SiteRegistrationActionData))]
+        public void TestAllowableStatusChange_Archive(SiteRegistrationAction action)
```

Comment:
To fit the existing convention, should be `TestAllowableStatusChange_Editable`

Reply:
Correct. I just renamed the test methods to make more sense.